mtp_noms_ops/apps/security/templatetags/security.py

```diff
@@ -246,6 +246,17 @@ def format_security_check_status(check):
         return security_check_statuses.get(check.get('status'), check.get('status'))
 
 
+@register.filter
+def format_security_check_rejection_reasons(check):
+    if not check or not check.get('rejection_reasons'):
+        return ''
+    return format_html_join(
+        mark_safe('<br />\n'),
+        '{}',
+        map(lambda description: [description], human_readable_check_rejection_reasons(check['rejection_reasons']))
```

Comment:
This is the first time I see `format_html_join()` but from what I can tell from [the Django docs](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.html.format_html_join) it's a fancy way of joining items in a list.

Maybe it's a stupid question but why the `map` to map descriptions to lists with 1 description? Is the `map` there because `format_html_join()` wants a generator?

Reply:
`human_readable_check_rejection_reasons` returns an iterable of strings (`list[str]`) but `format_html_join` needs an iterable _of iterables_ of stuff (e.g. `list[list[Any]]`).

this function effectively works by calling `'{}'.format(*item)` for each item in the last parameter. so if the last parameter is a list of strings, the `*` expands each string into characters

Reply:
would this be clearer?

```python
(
    (description,)
    for description in human_readable_check_rejection_reasons(check['rejection_reasons'])
)
```